src/content/whats-new/2024/04/whats-new-04-12-get-chart-link-rbac.md

```diff
@@ -0,0 +1,33 @@
+---
+title: 'Important permission update for sharing charts'
+summary: 'Update gives you more control over who can use the Get chart link feature'
+releaseDate: '2024-04-12'
+learnMoreLink: 'https://docs.newrelic.com/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard/#data-share'
+getStartedLink: ''
+---
+
+We've made improvements to our existing Get Chart Link feature, which will take effect on **April 30th, 2024.**
+
+## What's changing?
+
+We've improved Role-Based Access Controls (RBACs) for Pro and Enterprise accounts. 
+
+If you're a user from **Pro or Enterprise account, you can expect these changes:
+
+- Admins receive new [permission options](https://docs.newrelic.com/docs/accounts/accounts-billing/new-relic-one-user-management/user-permissions/#dashboards) to allow certain users to view, create, and revoke chart links.
+- Other users gain access to these permissions via admins. The access is only possible only custom roles created  by admins.
+- Your existing links within your account will continue to work. If you want to delete or disable any of these existing links, review the Data share section of our [Manage Your Dashboard Documentation](https://docs.newrelic.com/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard/#data-share) documentation.
+
+## What action is needed by you?
```

Comment:
@louisleungk Can we provide a little bit more detail about what high level actions need to be taken? For example, 

- Update your user permissions to X
- What changes need to be made to share data?
- Revoke live chart URLs from X users at [blahblahblah]